sdk/Sdk/Tasks/EnhanceExtensionsMetadata.cs

```diff
@@ -27,7 +26,7 @@ public override bool Execute()
             var extensionsMetadata = JsonSerializer.Deserialize<ExtensionsMetadata>(json);
             ExtensionsMetadataEnhancer.AddHintPath(extensionsMetadata?.Extensions ?? Enumerable.Empty<ExtensionReference>());
 
-            string newJson = JsonSerializer.Serialize(extensionsMetadata);
+            string newJson = JsonSerializer.Serialize(extensionsMetadata, _serializerOptions);
```

Comment:
Minor change to make the extensions.json output a little nicer to read.

Reply:
Great... I noticed that wasn't formatted :)